packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -2574,11 +2628,15 @@ export class WalletService {
         },
         (next) => {
           if (opts.validateOutputs === false) return next();
-          const validationError = this._validateOutputs(opts, wallet, next);
-          if (validationError) {
-            return next(validationError);
+          if (!Constants.UTXO_COINS[wallet.coin.toUpperCase()]) {
+            next();
```

Comment:
dont you think some output validation should be done?

like addr / amount on them?  Remember BWS is an open api that could be exploited.